tests/mock_module/service.py

```diff
@@ -1,12 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
 """Example service."""
 
 from invenio_records_resources.services import RecordService, \
     RecordServiceConfig
+from invenio_records_resources.services.records.search import terms_filter
```

Comment:
minor: @lnielsen  commented at some point about Elasticsearch DSL `TermsFilter`. However, I cannot find it in their docs now... 🤔